setup.py

```diff
@@ -15,12 +15,12 @@
     # command again and the install succeeds because absl-py has been
     # installed. This is stupid, but harmless.
     'dm_control @ https://api.github.com/repos/deepmind/dm_control/tarball/92f9913013face0468442cd0964d5973ea2089ea',  # noqa: E501
-    'gym[all]==0.10.11',
+    'gym[all]==0.12.0',
```

Comment:
why not 0.12.1?

Reply:
0.12.1 was throwing off all the tests because of a change in the way a property was exposed. Still looking into it. Will do that before submitting

Reply:
i think they changed from using properties to member variables for `observation_space` and `action_space`. this can create problems when an inheriting class tries to use a property.

tldr: you will probably have to change some properties to member variables in garage

Reply:
we might as well just do it now